### PR TITLE
Add option to parametrize checks

### PR DESCRIPTION
**Pull request by damjanek**

### CODE REVIEW

**Comment by cloudlab-autobuild** [comment]
Can one of the admins verify this patch?

File: linux/map.jinja [hunk 1]
```diff
@@ -185,3 +185,28 @@
          },
     },
 }, grain='os_family', merge=salt['pillar.get']('linux:storage')) %}
+
+
+{% set monitoring = salt['grains.filter_by']({
```
Comment (fpytloun) [inline linux/map.jinja]: You should decide if you want to have defaults in map.jinja or use `.get()` in support metadata, it's not needed to have both.

Comment (damjanek) [inline linux/map.jinja]: {%- from "linux/map.jinja" import system,network,storage,monitoring with context -%} - is it not going to blow if I get rid of map.jinja entry for this?

Comment (fpytloun) [inline linux/map.jinja]: You would need to refer `pillar.linux.get('monitoring')` in support meta.
It's ok to use map.jinja for it, just move the defaults there so you don't need to use .get('warn', sumthin)

Comment (damjanek) [inline linux/map.jinja]: Fixed. I'm sticking with map.jinja

Comment (fpytloun) [inline linux/map.jinja]: and you don't need these filters per distribution, use 'default' only

**Pull request merged**